bin/main.py

```diff
@@ -19,6 +19,5 @@
     except ImportError:
         pass
 
-from tutor.commands.cli import main
```

Comment:
There is a reason that this import is placed after plugins are loaded. I guess I should have written that down at the time because I can't remember that reason at all... 

Ah I get it! At the time the plugin commands were being added to the cli object during module import: https://github.com/overhangio/tutor/blob/159e24a95d52826f3072261d3b96d3f4247e931a/tutor/commands/cli.py#L64
The plugin commands are now added as part of the `tutor.commands.cli.main()` function call. So your change should be fine and is indeed a good idea :+1: